util/base/src/main/java/jetbrains/jetpad/base/platform/Platform.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2012-2017 JetBrains s.r.o
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package jetbrains.jetpad.base.platform;
+
+public final class Platform {
+  private static PlatformType ourPlatform;
+
+  public static PlatformType getPlatform() {
+    if (ourPlatform == null) {
```

Comment:
May be we need to return "unknown" by default? It seems that it will be difficult to writes tests with such approach.